device_doctor/lib/src/config.dart

```diff
@@ -0,0 +1,36 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:meta/meta.dart';
```

Comment:
Do we actually need this class?  we can potentially directly create the classes in the factory method using the flags.

Reply:
Based on our current logic, this config class is not needed. Updated to use flag directly as suggested.